test-suite/Makefile

```diff
@@ -57,11 +56,22 @@ export CAML_LD_LIBRARY_PATH:=$(shell echo $(COQPREFIX)/lib/stublibs):$(CAML_LD_L
 
 # COQLIB is an env variable so no quotes
 COQLIB?=
+
 ifeq ($(COQLIB),)
+# This should guard againt the COQLIBINSTALL directory not existing,
+# however, it doesn't work, defying my knowledge about makefiles.
+ifneq ($(wildcard $(COQLIBINSTALL)),)
```

Comment:
COQLIBINSTALL doesn't exist because it is quoted

Reply:
Oh O_O, ok, hack added.